src/faas/v0/context.ts

```diff
@@ -85,6 +85,20 @@ export abstract class AbstractRequest {
   protected constructor(data: string | Uint8Array) {
     this.data = data;
   }
+
+  text(): string {
+    const stringPayload = typeof this.data === 'string' 
+      ? this.data 
+      : new TextDecoder('utf-8').decode(this.data);
+
+    // attempt to deserialize as a JSON object
```

Comment:
The comment looks like an accidental copy/paste

Reply:
done